src/main/java/org/weasis/dicom/google/explorer/DownloadManager.java

```diff
@@ -118,20 +117,12 @@ public void addSelectionAndnotify(File[] file) {
                                 || MimeInspector.isMatchingMimeTypeFromMagicNumber(file1, DicomMediaIO.MIMETYPE)) {
                             DicomMediaIO loader = new DicomMediaIO(file1);
                             if (loader.isReadableDicom()) {
-                                if (dicomSeries == null) {
-                                    dicomSeries = loader.getMediaSeries();
-                                } else {
-                                    dicomSeries.addAll(Arrays.asList(loader.getMediaElement()));
-                                }
+                                ViewerPluginBuilder.openSequenceInDefaultPlugin(loader.getMediaSeries(), dicomModel, false, false);
```

Comment:
How do you view what the parameters mean? I see that the method has gone from taking true to false but I am not sure what it means....

Also, just to clarify, do other parts in Weasis also load each instance in a separate tab if you open a study?